### PR TITLE
Login Features and Landing page UI

### PR DESCRIPTION
**Pull request by jeraldbatal22**
This PR add some Navbar UI, Landing Page UI, Login features, if user is not approve by the admin gives error validation and if the user is approve he receive an confirmation email to his/her gmail account and also delete some unnecessary files.

image
image
image

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -61,3 +66,14 @@ form {
...
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+/* Firefox */
+input[type=number] {
+  -moz-appearance: textfield;
+}
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/application.css]: add new line at the end of file

File: app/views/user_mailer/confirmation_email.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<h1>Welcome to the Stock Trading App, <%= @user.email %></h1>
-<p>You have successfully approved your signup. Thankyou for joining Stock Trading App <%=@user.username%></p>
+<h1>Welcome to the Stock On You App, <%= @user.email %></h1>
+<p>You have successfully approved your signup. Thankyou for joining Stock On You App <%=@user.username%></p>
 <p>You can now signin your credentials. Enjoy and have a great day :) </p>
```
Comment (analyn-cajocson) [inline app/views/user_mailer/confirmation_email.html.erb]: add new file at the end of file

File: db/migrate/20210916124318_devise_create_users.rb [hunk 3]
```diff
@@ -30,10 +30,10 @@ def change
...
 
       ## Confirmable
-      t.string   :confirmation_token
-      t.datetime :confirmed_at
-      t.datetime :confirmation_sent_at
-      t.string   :unconfirmed_email # Only if using reconfirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
```
Comment (analyn-cajocson) [inline db/migrate/20210916124318_devise_create_users.rb]: do not include these changes. You already have a migration file for this

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (jeraldbatal22) [reply app/assets/stylesheets/application.css]: done na po

File: app/views/user_mailer/confirmation_email.html.erb (hunk 2 above)
Comment (jeraldbatal22) [reply app/views/user_mailer/confirmation_email.html.erb]: done

File: db/migrate/20210916124318_devise_create_users.rb (hunk 3 above)
Comment (jeraldbatal22) [reply db/migrate/20210916124318_devise_create_users.rb]: done

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**